src/test/java/io/stargate/sgv2/jsonapi/api/v1/FindIntegrationTest.java

```diff
@@ -409,6 +409,31 @@ public void inConditionNonIdField() {
           .body("data.documents[0]", jsonEquals(expected1));
     }
 
+    @Test
+    public void inConditionNonIdFieldMulti() {
+      String json =
+          """
+              {
+                "find": {
+                    "filter" : {
+                         "username" : {"$in" : ["user1", "user2"]}
+                    }
+                  }
+              }
+              """;
+      given()
+          .header(HttpConstants.AUTHENTICATION_TOKEN_HEADER_NAME, getAuthToken())
+          .contentType(ContentType.JSON)
+          .body(json)
+          .when()
+          .post(CollectionResource.BASE_PATH, namespaceName, collectionName)
+          .then()
+          .statusCode(200)
+          .body("data.documents", hasSize(2))
```

Comment:
Can we also test that the document ids are what we expect? Or at least test that usernames are "user1" and "user2"?

Reply:
sure. Thanks.